CMakeLists.txt

```diff
@@ -30,8 +30,12 @@ cmake_minimum_required(VERSION 2.8)
 # Using c++11;
 # Optimization on architecture
 #-------------------------------------------------------------------------------
+if(NOT WIN32)
 add_definitions("-Wall -Wno-sign-compare -O3 -std=c++11 
 -march=native -Wno-strict-aliasing -Wno-comment")
+else(WIN32)
+add_definitions("/WX- /MT")
```

Comment:
Does this definitions use the best compiler optimization for speed? Sorry, I 'm not a Windows developer.

Reply:
No，because gtest version is old, add  ``/WX-``  to avoid warning error.

Reply:
There is no option about ``/O3`` in VS, and it recommend ``/O2`` for release build.